dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterable.java

```diff
@@ -80,11 +81,35 @@ public class ZonedIntervalIterable implements Iterable<ZonedInterval> {
   @Nonnull
   public static ZonedIntervalIterable forConnectorArguments(@Nonnull ConnectorArguments arguments)
       throws MetadataDumperUsageException {
+    return ZonedIntervalIterable.forConnectorArguments(arguments, ChronoUnit.HOURS);
```

Comment:
I understand this was brought in by an earlier commit but this change shows that using a time unit is awkward because what we are really looking at is a time duration and we just deduce that duration from the unit by the convention of using 1 of the provided unit. How about moving to `Duration` (and limiting to durations d where d <= 1 day and 1 day / d is a natural number?

Reply:
Adding this in the `TODO` list

Reply:
Agree with @haselbach - do we have a precedent for parsing a duration somewhere?